tutorial/source/cvae.ipynb

```diff
@@ -303,6 +303,39 @@
     "```"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training\n",
+    "The training code can be found in the [Github repo](https://github.com/pyro-ppl/pyro/tree/dev/examples/cvae). \n",
+    "Click play in the video below to watch how the CVAE learns throughout approximately 40 epochs.\n",
+    "\n",
+    "<p>\n",
+    "    <video muted controls loop width=\"100%\">\n",
+    "        <source src=\"https://ucals-github.s3-sa-east-1.amazonaws.com/cvae_animation.mp4\" type=\"video/mp4\">\n",
+    "        <source src=\"https://ucals-github.s3-sa-east-1.amazonaws.com/cvae_animation.webm\" type=\"video/webm\">\n",
+    "    </video>\n",
```

Comment:
maybe try avoiding the indentation? Maybe nbsphinx misinterprets the spaces as a code block?